source/export-query-to-language.txt

```diff
@@ -10,8 +10,8 @@ Export Query to Specific Language
 
 You can export queries created in the :ref:`query bar
 <compass-query-bar>` to one of the supported languages; Java, Node,
-C#, and Python 3. This feature facilitates reformatting and utilizing
-queries created in |compass| for use in your application.
+C#, Python 3, Ruby, Go, Rust, and PHP. This feature facilitates reformatting 
+and utilizing queries created in |compass| for use in your application.
```

Comment:
[suggestion] "Utilizing" is a big word where a small one will do! Let's just say "use."

"This feature allows you to reformat and use |compass| queries in your application."

Reply:
Done.